src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -232,10 +234,18 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			if (this.cellModel.currentMode !== CellEditModes.WYSIWYG) {
 				needsTransform = false;
 			} else {
+				let result = undefined;
+				const isFile = URI.parse(linkCalloutResult?.insertUnescapedLinkUrl).scheme === 'file';
+				if (!path.isAbsolute(linkCalloutResult?.insertUnescapedLinkUrl) && isFile) {
+					const notebookDirName = path.dirname(this.cellModel?.notebookModel?.notebookUri.fsPath);
+					const relativePath = linkCalloutResult?.insertUnescapedLinkUrl.replace(/(\.\\ | \.\/)/g, '/');
```

Comment:
Could you explain what this is supposed to be doing?

And also what about my previous comment : 

> This doesn't seem like it'd handle multiple .. well...for example if I have ..\..\..\notebook.ipynb then wouldn't relative path end up being ./././notebook.ipynb?

Is there something making it so that users can't enter in ..\ manually in the URL?

Reply:
The reason I replace the `..\` is because even if we have these two paths that should work. The second one doesn't work when we make the transformation to markdown. 
`../../../EmptyBook/readme.md`

`..\..\..\EmptyBook\readme.md`

Reply:
I'll just replace the backlashes for slashes instead of that regular expression